src/Marten.Testing/ScenarioCopyAndReplaceStream.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Linq;
+using Marten.Events;
+using Marten.Services;
+using Marten.Testing.Events;
+using Marten.Testing.Events.Projections;
+using Xunit;
+
+namespace Marten.Testing
+{
+	public class ScenarioCopyAndReplaceStream : DocumentSessionFixture<NulloIdentityMap>
+	{
+		public ScenarioCopyAndReplaceStream()
+		{
+			StoreOptions(options =>
+			{
+				options.Events.StreamIdentity = StreamIdentity.AsString;
+			});
+		}
+
+		[Fact]
```

Comment:
If this a sample for docs, can we remove/comment the `[Fact]` so that it does not get run as a unit test.

Reply:
@mysticmind: Certainly can be removed. Comes with the downside that should there be behavioural breaking changes (in this case something in the execution path leading to an exception, which is not likely :), the sample breaks & goes unnoticed.

Reply:
@jokokko okay, make sense :-)